### PR TITLE
Add eth namespace methods for specificity

### PR DESCRIPTION
**Pull request by franklywatson**
To help integrators and developers am adding the supported methods, and those we plan to support, for added clarity when they are testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,58 @@ FlowEVM Gateway implements the Ethereum JSON-RPC API for the [FlowEVM](https://d
 
 FlowEVM Gateway is compatible with the majority of standard Ethereum JSON-RPC APIs allowing seamless integration with existing Ethereum-compatible web3 tools via HTTP. FlowEVM Gateway honors Ethereum's JSON-RPC namespace system, grouping RPC methods into categories based on their specific purpose. Each method name is constructed using the namespace, an underscore, and the specific method name in that namespace. For example, the `eth_call` method is located within the `eth` namespace.
 
-Listed below are the JSON-RPC namespaces currently supported by the FlowEVM Gateway:
+Listed below are the JSON-RPC namespaces and methods currently supported by the FlowEVM Gateway:
 
 * `eth`
+  * Supported
```
Comment (sideninja) [inline README.md:14]: Can we add this at the bottom and just link it here.

Comment (franklywatson) [reply README.md:14]: Sure

File: README.md [hunk 2]
```diff
@@ -8,11 +8,58 @@ FlowEVM Gateway implements the Ethereum JSON-RPC API for the [FlowEVM](https://d
...
+    * eth_newFilter
+    * eth_uninstallFilter
+    * eth_getFilterLogs
+    * eth_getFilterChanges
+    * eth_newBlockFilter
+    * eth_newPendingTransactionFilter
 * `web3`
+  * Supported
+    * web3_clientVersion
+    * sha3
```
Comment (m-Peter) [inline README.md:57]:
```suggestion
    * web_sha3
```

File: README.md [hunk 3]
```diff
@@ -8,11 +8,58 @@ FlowEVM Gateway implements the Ethereum JSON-RPC API for the [FlowEVM](https://d
...
+    * eth_getTransactionByBlockHashAndIndex
+    * eth_getBlockByHash
+    * eth_getBlockReceipts
+    * eth_getBlockTransactionCountByHash
+    * eth_getBlockTransactionCountByNumber
+    * eth_accounts
+    * eth_sign
+    * eth_signTransaction
+    * eth_sendTransaction
+    * eth_getUncleCountByBlockHash // return empty
```
Comment (m-Peter) [inline README.md:38]:
```suggestion
    * eth_getUncleCountByBlockHash // returns 0
```

File: README.md [hunk 4]
```diff
@@ -8,11 +8,58 @@ FlowEVM Gateway implements the Ethereum JSON-RPC API for the [FlowEVM](https://d
...
+    * eth_gasPrice 
+    * eth_getBalance
+    * eth_estimateGas
+    * eth_getTransactionByBlockNumberAndIndex
+    * eth_getTransactionByBlockHashAndIndex
+    * eth_getBlockByHash
+    * eth_getBlockReceipts
+    * eth_getBlockTransactionCountByHash
+    * eth_getBlockTransactionCountByNumber
+    * eth_accounts
```
Comment (m-Peter) [inline README.md]: These 4:
- `eth_accounts`
- `eth_sign`
- `eth_signTransaction`
- `eth_sendTransaction`

are also unsupported as described in https://github.com/onflow/flow-evm-gateway/issues/8 . They can be called, but they simply return an error, as described in the issue. So maybe we should move them to unsupported?

**Review by m-Peter — APPROVED** [review]
Nice :ok_hand:

**Review by sideninja — APPROVED** [review]

**Pull request merged**